src/pytest_benchmark/table.py

```diff
@@ -38,11 +39,23 @@ def display(self, tr, groups, progress_reporter=report_progress):
                         benchmarks, tr, "{line} ({pos}/{total})", line=line))
                     best[prop] = max(bench[prop] for _, bench in progress_reporter(
                         benchmarks, tr, "{line} ({pos}/{total})", line=line))
+                    try:
+                        baseline[prop] = max(bench[prop] for _, bench in progress_reporter(
+                            benchmarks, tr, "{line} ({pos}/{total})", line=line)
+                            if bench.get("baseline", True))
+                    except ValueError:
```

Comment:
Can we avoid the try/except somehow? What actually can raise that error?

Reply:
If there is no benchmark in the group marked as `baseline` this will end up calling `max(())` which raises *ValueError: max() argument is empty sequence*.

Should I convert this to an `if`? It would require evaluating the array of values up-front, then check their `len(…)`. Or do you prefer me adding a comment about this?